incubator/kubevirt/README.md

```diff
@@ -0,0 +1,61 @@
+```
+$ helm install --name my-release incubator/kubevirt --set master.ip=10.10.10.10,master.selector=master.test.com
+```
+
+After installation finish, it better to remove extra permissions from the system.
+
+```
+$ kubectl delete clusterrolebinding kube-system-cluster-admin
+```
+
+## Deleting the Charts
+
+Delete the Helm deployment as normal
+
+```
+$ helm delete my-release
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the kubevirt chart and their default values.
+
+|                Parameter                  |            Description              |          Default           |
+| ----------------------------------------- | ------------------------------------| -------------------------- |
+| `master.ip`                               |k8s master IP                        | `10.10.10.10`              |
+| `master.hostname`                         |k8s master hostname                  | `master.test.com`          |
+| `haproxy.repository`                      |haproxy container name               | `kubevirt/haproxy`         |
+| `haproxy.tag`                             |haproxy container tag                | `latest`                   |
+| `libvirt.repository`                      |libvirt container name               | `kubevirt/libvirt`         |
+| `libvirt.tag`                             |libvirt container tag                | `latest`                   |
+| `spiceproxy.repository`                   |spice-proxy container name           | `kubevirt/spice-proxy`     |
+| `spiceproxy.tag`                          |spice-proxy container tag            | `latest`                   |
+| `virtapi.repository`                      |virt-api container name              | `kubevirt/virt-api`        |
+| `virtapi.tag`                             |virt-api container tag               | `latest`                   |
+| `virtcontroller.repository`               |virt-controller container name       | `kubevirt/virt-controller` |
+| `virtcontroller.tag`                      |virt-controller container tag        | `latest`                   |
+| `virtcontroller.virtlauncher.repository`  |virt-launcher container name         | `kubevirt/virt-launcher`   |
+| `virtcontroller.virtlauncher.tag`         |virt-launcher container tag          | `latest`                   |
+| `virtcontroller.virtmigration.repository` |virt-migrator container name         | `kubevirt/virt-migrator`   |
+| `virtcontroller.virtmigration.tag`        |virt-migrator container tag          | `latest`                   |
+| `virthandler.repository`                  |virt-handler container name          | `kubevirt/virt-handler`    |
+| `virthandler.tag`                         |virt-handler container tag           | `latest`                   |
```

Comment:
Maybe collapsing all these repository and tag variables to one makes sense for the beginning. I don't have any strong opinion regarding to that. Right now we always release everything together. So if there is a bugfix in virt-handler which pushes  the release version, we do a now tag for all components. Therefore having the possibility to set an overall repostory base e.g. (kubevirt.repo=kubevirt) and an overall version (kubevirt.tag=latest) might be a good start.

Reply:
So at the end, the following set of variables should be sufficient:

```
kubevirt.repo=kubevirt
kubevirt.tag=latest
kubevirt.apiserver.externalip=10.10.10.10
kubevirt.spiceproxy.externalip=10.10.10.10
```